Mjml.Net/Components/Body/SpacerComponent.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace Mjml.Net.Components.Body
+{
+    public sealed class SpacerComponent : BodyComponentBase
+    {
+        public override string ComponentName => "mj-spacer";
+
+        public override AllowedAttributes? AllowedAttributes { get; } =
+            new AllowedAttributes
+            {
+                ["border"] = AttributeType.String,
+                ["border-bottom"] = AttributeType.String,
+                ["border-left"] = AttributeType.String,
+                ["border-right"] = AttributeType.String,
+                ["border-top"] = AttributeType.String,
+                ["container-background-color"] = AttributeType.Color,
+                ["height"] = AttributeType.PixelsOrPercent,
+                ["padding"] = AttributeType.PixelsOrPercent,
```

Comment:
I have a default value of `10px 25px` on the button. We will need to handle this.

Reply:
I have something new for the attributes. Will be published soon.